sdlib/d/gc/tcache.d

```diff
@@ -42,7 +42,8 @@ public:
 		if (isSmallSize(asize)) {
 			auto ptr = arena.allocSmall(emap, asize);
 			auto pd = getPageDescriptor(ptr);
-			setSmallUsedCapacity(pd, ptr, size);
+			auto si = SlabAllocInfo(pd, ptr);
+			si.setUsedCapacity(size);
```

Comment:
Just change setSmallUsedCapacity and revert all these changes. You can do them in them next step.